### PR TITLE
Added section how to checkout from SCM via m2e plugin in eclipse (omitting command line)

### PR DESCRIPTION
**Pull request by romeokienzler**
## What changes were proposed in this pull request?

Added section how to checkout from SCM via m2e plugin in eclipse (omitting command line)

## How was this patch tested?

Visually reviewed in fork

## Quick checklist

The following checklist helps ensure your PR is complete:
=> omitted, only gh-pages

- [ ] Reviewed the [Contributing Guidelines](https://github.com/deeplearning4j/deeplearning4j/blob/master/CONTRIBUTING.md) and followed the steps within.
- [ ] Created tests for any significant new code additions.
- [ ] Relevant tests for your changes are passing.
- [ ] Ran mvn formatter:format (see [formatter instructions](http://code.revelc.net/formatter-maven-plugin/examples.html#Setting_Source_Files) for targeting your specific files).

### CODE REVIEW

File: gettingstarted.md [hunk 1]
```diff
@@ -60,6 +60,28 @@ If you use Eclipse, you will need to install the Maven plugin for Eclipse: [ecli
 
 Michael Depies has written this guide to [installing Deeplearning4j on Eclipse](https://depiesml.wordpress.com/2015/08/26/dl4j-gettingstarted/).
 
+## <a name="eclipse_mvn_scm_checkout">Using DL4J Examples in Eclipse with direct checkout from GIT via the Maven plugin</a> 
+
+If you just want to get the examples running within Eclipse and without using command line at all you can also directly checkout the project from SCM:
```
Comment (huitseeker) [inline gettingstarted.md]: Maybe expand "from Eclipse's built-in Source Control Management (SCM):" once

Comment (romeokienzler) [inline gettingstarted.md]: @huitseeker what do you mean by "expand "from Eclipse's built-in Source Control Management (SCM):" once" ?

Comment (huitseeker) [inline gettingstarted.md]: I mean introduce the acronym by saying what it stands for on the 1st occurrence
http://www.americanwritingandediting.se/swenglish.html#acronyms

**Comment by romeokienzler** [comment]
@huitseeker expanded: ...from Eclipse's built-in Source Control Management (SCM), let me know if something else prevents you from merging, I'll fix...

**Pull request merged**